src/app/pages/TopicPage/index.test.jsx

```diff
@@ -32,14 +51,15 @@ describe('A11y', () => {
   });
 
   it('should render an unordered list when there is more than one promo', () => {
-    const { container, queryByRole } = render(
-      <TopicPageWithContext
-        pageData={pidginMultipleItems}
-        lang="pcm"
-        service="pidgin"
-      />,
-    );
+    const { container, queryByRole } = render(<TopicPageWithContext />);
     expect(queryByRole('list')).toBeInTheDocument();
     expect(container.getElementsByTagName('li').length).toEqual(4);
   });
+
+  it('should show ads when enabled', () => {
```

Comment:
Thanks for adding the tests, we could do with independently checking:
- When ads are enabled but not in a location to show ads
- When ads are disabled, they do not show